README.md

```diff
@@ -1,36 +1,27 @@
-# Welcome to your [FASTN site](https://fastn.io/)
+Poppins: fastn Font Package
 
-FTD Font Package Template Repo
+This repository contains a [fastn font package](https://fpm.dev/featured/fonts/) containing [Google Font: 
+Poppins](https://fonts.google.com/specimen/Poppins/about).
 
+Geometric sans serif typefaces have been a popular design tool ever since these actors took to the world’s stage. Poppins is one of the new comers to this long tradition. With support for the Devanagari and Latin writing systems, it is an internationalist take on the genre.
```

Comment:
```suggestion
Geometric sans serif typefaces have been a popular design tool ever since these actors took to the world’s stage. Poppins is one of the new comers to this long tradition. With support for the Devanagari and Latin writing systems, it is an internationalist take on the genre.
```